www/class/config-generate-remote/ContactGroup.php

```diff
@@ -48,6 +48,8 @@ class ContactGroup extends AbstractObject
     protected $stmtContact = null;
     protected $stmtCgService = null;
 
+    protected $illegalCharsTab = array();
```

Comment:
```suggestion
```

Reply:
There are no need of this property.
The logic of sinitazing could be put in a method.

Reply:
> There are no need of this property. The logic of sinitazing could be put in a method.

I added this property to have a cache of the SQL request, in order to avoid querying the database for each name to sanitize (especially since this process is not on the input of a unique value, but when the entire engine configuration is generated).